crates/bevy_ecs/src/event.rs

```diff
@@ -116,6 +116,8 @@ enum State {
 /// when events are cleared.
 /// This complicates consumption and risks ever-expanding memory usage if not cleaned up,
 /// but can be done by adding your event as a resource instead of using [`AppBuilder::add_event`].
+///
+/// [`AppBuilder::add_event`]: https://docs.rs/bevy/*/bevy/ecs/system/struct.EntityCommands.html
```

Comment:
I'm not sure I understand the formatting you're aiming for with this link. It's a useful link though, even if I think it needs more context!

Reply:
This link will break in the future as it uses * as version.

Reply:
Yep the idea was to link back to the bevy crate, but as bjorn3 stated the problem with this is that the link might break in the future. Using the exact version isnt really too nice either though as it will get outdated over time 😕

Reply:
Isn't this type also just `[crate::system::EntityCommands]`? We're going to add some `html_root_url` attributes to the bevy docs at some point (probably using the [`version-sync`](https://crates.io/crates/version-sync) crate), which would hopefully remove the need for the `bevy/...` folders entirely, in favour of `bevy_ecs/...`

Reply:
Oh I just realised the link is wrong, its supposed to link to `https://docs.rs/bevy/*/bevy/app/struct.AppBuilder.html#method.add_event` 😅